src/environment/unix.go

```diff
@@ -65,15 +65,15 @@ func (env *ShellEnvironment) Platform() string {
 		env.Cache().Set(key, platform, -1)
 	}()
 	if wsl := env.Getenv("WSL_DISTRO_NAME"); len(wsl) != 0 {
-		platform = strings.ToLower(wsl)
-		return strings.Split(platform, "-")[0]
-	}
-	platform, _, _, _ = host.PlatformInformation()
-	if platform == "arch" {
-		// validate for Manjaro
-		lsbInfo := env.FileContent("/etc/lsb-release")
-		if strings.Contains(strings.ToLower(lsbInfo), "manjaro") {
-			platform = "manjaro"
+		platform = strings.Split(strings.ToLower(wsl), "-")[0]
```

Comment:
Can you do an early return here rather than use an else statement?

Reply:
Updated.